cloud: update backup policies and execution rules (#22227)
pingcap/docs: tidb-cloud/premium/backup-and-restore-premium.md

## Automatic backups

> {{{ .premium }}} offers enhanced automatic backup capabilities, combining high-frequency snapshots with log backups to ensure maximum data reliability for production environments.

Reviewer: ```suggestion
{{{ .premium }}} offers enhanced automatic backup capabilities, combining high-frequency snapshots with log backups to maximize data reliability for production environments.
```

### Automatic backup policies

{{{ .premium }}} instances provide a comprehensive data protection strategy through a multi-layered backup architecture:

- Point-in-Time Recovery (PITR)
    - Retention: 7 days.

>     - Restore detail: allows restoration to any specific moment within the 7-day retention period.

Reviewer: ```suggestion
- Point-in-Time Recovery (PITR)
    - Retention period: 7 days.
    - Restore granularity: supports restoration to any specific moment within the 7-day retention window.
```

>     - Restore Detail: Allows restoration to any specific moment within the 7-day retention period.

Reviewer: ```suggestion
    - Restore detail: allows restoration to any specific moment within the 7-day retention period.
```

> - **Hourly backup snapshot**

Reviewer: ```suggestion
- Hourly backup snapshot
```

    - Retention: 7 days.

>     - Restore detail: regular restoration is available for each hourly snapshot generated within the 7-day window.

Reviewer: ```suggestion
- Hourly backup snapshot
    - Retention period: 7 days.
    - Restore granularity: supports restoration to any specific hourly snapshot generated within the 7-day retention window
```

> - **Daily backup snapshot**

Reviewer: ```suggestion
- Daily backup snapshot
```

## Automatic backups

{{{ .premium }}} offers enhanced automatic backup capabilities, combining high-frequency snapshots with log backups to maximize data reliability for production environments.

### Automatic backup policies

> {{{ .premium }}} instances provide a comprehensive data protection strategy through a multi-layered backup architecture:

Reviewer 2: ```suggestion
{{{ .premium }}} instances use a multi-layer backup architecture to protect your data, as described in the following table:
```

- Point-in-Time Recovery (PITR)

>     - Retention period: 7 days.

Reviewer 2: ```suggestion
| Backup type | Retention period | Restore granularity |
| --- | --- | --- |
| **Point-in-time recovery (PITR)** | 7 days | Restore to any specific point in time within the 7-day window. |
| **Hourly snapshot** | 7 days | Restore from any hourly snapshot generated within the last 7 days. |
| **Daily snapshot** | 33 days | Restore from any daily snapshot generated within the last 33 days. By default, daily snapshots are captured at 00:00 UTC. |
```
Replcae L39-49 with a table with a table to make it easier to compare backup type.

    - Restore granularity: supports restoration to any specific moment within the 7-day retention window.

- Hourly backup snapshot
    - Retention period: 7 days.
    - Restore granularity: supports restoration to any specific hourly snapshot generated within the 7-day retention window

- Daily backup snapshot
    - Retention period: 33 days.
    - Restore granularity: supports restoration to any daily snapshot (captured at 0:00 UTC by default) within the 33-day retention window.

### Backup execution rules

> - Backup cycle: {{{ .premium }}} instances support both daily and hourly automatic backups.

Reviewer 2: ```suggestion
- **Backup cycle**: {{{ .premium }}} instances perform both hourly and daily automatic backups.
```

- Backup schedule:
    - Hourly backup: always runs at the start of every hour (on the hour).
    - Daily backup: executed daily at 0:00 UTC.

>     - Currently, backup time management or customization is not supported.

Reviewer 2: ```suggestion
    - Hourly backups run at the start of every hour.
    - Daily backups run at 00:00 UTC each day.
    - Currently, you cannot customize or manage backup schedules.
```

## Automatic backups

TiDB Cloud Premium offers enhanced automatic backup capabilities, combining high-frequency snapshots with log backups to ensure maximum data reliability for production environments.

### Automatic Backup Policy Details
TiDB Cloud Premium instances provide a comprehensive data protection strategy through a multi-layered backup architecture:

- **Point-in-Time Recovery (PITR)**
    - Retention: 7 days.
    - Restore Detail: Allows restoration to any specific moment within the 7-day retention period.

- **Hourly Backup Snapshot**
    - Retention: 7 days.
    - Restore Detail: Regular restoration is available for each hourly snapshot generated within the 7-day window.

- **Daily Backup Snapshot**
    - Retention: 33 days.

>     - Restore Detail: Restoration is available for daily snapshots (defaulted at 0:00 UTC) for the 33-day retention period.

Reviewer: ```suggestion
- **Daily backup snapshot**
    - Retention: 33 days.
    - Restore detail: restoration is available for daily snapshots (at 0:00 UTC by default) for the 33-day retention period.
```

### Backup Execution Rules
- **Backup Cycle**: Premium instances support both daily and hourly automatic backups.

- **Backup Time**:
    - Daily Backup: Runs at a fixed time of 0:00 UTC. Backup time management/customization is currently not supported.

>     - Hourly Backup: Always runs on the hour.

Reviewer: ```suggestion
- **Backup Time**:
    - Daily backup: runs at a fixed time of 0:00 UTC. Backup time management or customization is currently not supported.
    - Hourly backup: always runs on the hour.
```
